app/src/main/java/amazingme/activities/app/UserProfileActivity.java

```diff
@@ -28,7 +32,8 @@ public void bindToUserInterface() {
         nextBtn = (Button) findViewById(R.id.user_profile_next_button);
         firstName = (EditText) findViewById(R.id.user_profile_first_name_edit_text);
         lastName = (EditText) findViewById(R.id.user_profile_last_name_edit_text);
-        parentEmail = (EditText) findViewById(R.id.user_profile_parent_email_edit_text);
+        label = (TextView) findViewById(R.id.user_profile_parent_profile_label);
+        profile = (QuickContactBadge) findViewById(R.id.user_profile_parent_portrait);
```

Comment:
we need to see if this is what we need for a picture... 

we need to be able to set it and i'm not sure if this is what we will use... but let's keep it for now. good job putting it in there!